dotCMS/src/main/java/com/dotmarketing/db/commands/UpsertCommand.java

```diff
@@ -143,11 +147,19 @@ public void execute(DotConnect dotConnect, String query, Object... parameters) t
      * ON CONFLICT (conditionalColumn) DO UPDATE SET column1=value1, column2=value2, etc...
      */
 
+    private static final float POSTGRES_UPSERT_MINIMUM_VERSION = 9.5f;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Upper-case this literal "f" suffix. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS818)